ui/scripts/vpc.js

```diff
@@ -15,6 +15,53 @@
 // specific language governing permissions and limitations
 // under the License.
 (function($, cloudStack) {
+    var isNumeric = function (n) {
+        return !isNaN(parseFloat(n));
+    };
+    var createSafeCsvValue = function(value){
+        if(value){
+            return '"' + value + '"';
```

Comment:
Hi @rafaelweingartner, Thanks for adding this functionality.
Here, Will the input look like this: ("a","b","c"...) or (a,b,c...). The variable csv in generateCsvForAclRules function converts to string when it is initialized (concatenated with a comma string).

Reply:
@nitin-maharana thanks for the review, but I did not quite understand your question.
Here is an example of CSV generated using the code of this PR:
```
"id","#Rule","CIDR","Action","Protocol","#Protocol","Start Port","End Port","ICMP Type","ICMP Code","Traffic Type","Reason"
"ef06c57e-dc8b-40d9-a461-5a30072384ab","2","19.168.56.0/12","Allow","all",,,,,,"Ingress",
"45835dfb-260c-461d-8051-444ac764151e","102","19.168.56.0/12","Allow","tcp",,"1","1",,,"Ingress",
```

I am using this `createSafeCsvValue ` to enclose the value with double quotes, which will avoid problems with values that may have commas in them. If you open the file generated with this code in Excel, it is transparent for users.